### ISSUE TITLE
2.1.18 setting of BOS for number of winners seems broke

### ISSUE THREAD

**Issue by heneksj** [geoffhumphrey/brewcompetitiononlineentry#1117]
Version: 2.1.18

Installation URL:
https://turkeyshoot.brewcompetition.com/

Is your installation hosted on brewcompetition.com or brewcomp.com? yes

Description of Issue:
So I want to send the first place winner from each table to the BOS table, but want to award 1st second and third BOS prizes.  I set winners only to BOS table--because if I don't do that the pull sheets are for 1st, 2nd and 3rd on each table.   I assign first, second and third BOS winners.  But when you do that, only the first place winner shows up as a winner on the results page.  I have to then go back to assign 1st, 2nd, and 3rd to BOS in order to display properly.  Seems to me, you should just display anything that has a place entered.  Don't recall having this issue before.  

For enhancements, please prepend "Enhancement - " to the title. For issues/bugs, please prepend the BCOE&M version number the title.

**Comment by ercousin** [comment]
This appears to be a duplicate of:

https://github.com/geoffhumphrey/brewcompetitiononlineentry/issues/1071

**Comment by geoffhumphrey** [comment]
Yes, this is a duplicate of #1071 and will be addressed in version 3.0 (conversion), targeted for late 2020.
